src/modules/vtol_att_control/standard.cpp

```diff
@@ -368,6 +368,12 @@ void Standard::fill_actuator_outputs()
 
 	_actuators_out_1->control[actuator_controls_s::INDEX_THROTTLE] = _pusher_throttle;
 
+	/* translate MC actuators to FW actuators */
+	if (_vtol_schedule.flight_mode == TRANSITION_TO_FW || _vtol_schedule.flight_mode == TRANSITION_TO_MC) {
+		_actuators_out_1->control[actuator_controls_s::INDEX_ROLL] = -_actuators_mc_in->control[actuator_controls_s::INDEX_YAW];
+		_actuators_out_1->control[actuator_controls_s::INDEX_PITCH] = _actuators_mc_in->control[actuator_controls_s::INDEX_THROTTLE] - 0.5f;
```

Comment:
I have seen with the quadranger that pitch (although it was mostly an increase) based on variable factors like wind or small cg changes it can also pitch down. This patch will actually apply the correction that the controller is asking for. Although it's not very elegant as it does not consider various parameters, it will apply this logic only for a few (crucial) seconds.

I will remove the roll and test fly this tomorrow. Logs will show if this has the same effect as it does now in sitl.